metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -317,6 +316,8 @@ This package installs the Metron Profiler %{metron_home}
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 %changelog
+* Thu Jan 19 2017 Justin Leet <justinjleet@gmail.com> - 0.3.0
```

Comment:
Hmm, you're right @dlyle65535, we do lose some capabilities by not having the changelog in there.  You know what would be cool, if we could generate the spec files as part of maven's `generate-source` and fill in the changelog with the information from git.  Anyway, best of both worlds kinda thing, but not effort that should be done as part of this PR.